test/fixtures/cvp_nodes.yaml

```diff
@@ -1,9 +1,15 @@
 - node: cvp
   username: CvpRacTest
   password: AristaInnovates
+  api_token: null
+  is_cvaas: False
```

Comment:
Revert this file and add appropriate parameter checks to test_cvp_api.py

Reply:
Changed this file